specs/nimble-blazor/blazor-net8-upgrade-hld.md

```diff
@@ -0,0 +1,88 @@
+# .NET 8 Support for Nimble/Spright Blazor
+
+## Problem Statement
+
+Currently our Blazor projects (Nimble/Spright) target .NET 6, which will reach [end-of-support on November 12, 2024](https://devblogs.microsoft.com/dotnet/dotnet-6-end-of-support/).
+
+Before that date, we should update our Blazor projects to target .NET 8, which will remain in-support until November 2026 ([more info on versions / support duration here](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core)).
+
+
+## Links To Relevant Work Items and Reference Material
+
+Design System work items:
+- [Feature 2812085: Nimble Blazor .NET 8 upgrade](https://dev.azure.com/ni/DevCentral/_workitems/edit/2812085)
+- [nimble#1667: Blazor .NET 8 support](https://github.com/ni/nimble/issues/1667)
+
+Related work items:
+- ASW: [Epic 2753078: ASW .NET 8 Migration](https://dev.azure.com/ni/DevCentral/_workitems/edit/2753078)
+
+## Implementation / Design
+
+### Planned Changes ###
```

Comment:
There will be some doc updates needed too. If you search .md files for "net" you can see a few references to SDK versions in CONTRIBUTING, for example.

Reply:
Added a bullet point for doc updates.